ui/src/SpaceDashboard/DeleteSpaceForm.test.tsx

```diff
@@ -85,10 +85,10 @@ describe('Delete Space Form', () => {
         });
 
         describe('things to do', () => {
-            it('should call the space client when the leave and delete button is pressed with appropriate spaceId', () => {
+            it('should call the space client when the leave and delete button is pressed with appropriate spaceId', async () => {
                 SpaceClient.deleteSpaceByUuid = jest.fn(() => Promise.resolve());
-                act(() => {
-                    const bigRedButton = form.getByText('Delete space');
+                await act(async () => {
```

Comment:
Why was this change made? As far as I can tell this does not need to be async (`get*` query methods in Testing Library are synchronous).

Reply:
It originally was not, but there was a React (or React Testing) warning that I narrowed down to this test. This made that warning go away.